pandas/core/groupby/generic.py

```diff
@@ -1001,14 +1001,42 @@ def take(
         result = self._op_via_apply("take", indices=indices, axis=axis, **kwargs)
         return result
 
-    @doc(Series.skew.__doc__)
     def skew(
         self,
         axis: Axis | lib.NoDefault = lib.no_default,
         skipna: bool = True,
         numeric_only: bool = False,
         **kwargs,
     ) -> Series:
+        """
+        Return unbiased skew within groups.\n\nNormalized by N-1.
```

Comment:
Instead of `\n\n`, I think you can add newlines. Alternatively, I think it would be fine to replace `.\n\nN` with `, n`.